### DISCUSSION TITLE
I can no longer get into the system (not even as admin)

### DISCUSSION THREAD

**Discussion by KevinKrueger** [open-webui/open-webui#1128] [General]
![image](https://github.com/open-webui/open-webui/assets/25382167/69460bee-1937-49d0-90d3-ebbad83806b8)

Nobody is logged in!

**Comment by tjbck** [comment]
Seems like you might've accidentally created a second account, if you don't have pre-existing chat logs you can just delete the `webui.db` file and you'll be able to create a new admin account.

**Reply by KevinKrueger** [reply]
Ok.

But that must not be the solution!

What if I want 10 users to use it? Should they then create a new account each time after I delete the file?

**Reply by justinh-rahb** [reply]
The issue is that 1) only the first account created is an admin and 2) subsequent accounts created default to pending status and must be activated by an admin. Therefore, to get in this situation, you'd need to forget what your first account's password is. The solution is not forgetting your admin account password.

**Reply by tjbck** [reply]
All in our doc: https://docs.openwebui.com/getting-started/ :)

**Comment by Touffman78** [comment]
You can edit the webui.db file from CLI and change user's parameters/password
